docs/runbooks/CDINoDefaultStorageClass.md

```diff
@@ -2,37 +2,39 @@
 
 ## Meaning
 
-This alert fires when there is no default (Kubernetes or KubeVirt) storage
-class, and a data volume is pending for one.
+This alert fires when there is no default (Kubernetes or virtualization) storage
+class, and a data volume is `Pending` for one.
```

Comment:
Minor suggestions:
```
This alert fires when a data volume is `Pending` because 
there is no default (Kubernetes or virtualization) storage class.
```

Do you know if we _need_ to specify the "Kubernetes or virtualization" part? If we don't, it's much more readable, IMO:

```
This alert fires when a data volume is `Pending` because 
there is no default storage class.
```

Reply:
Since "virtualization default storage class" is quite new, users implicitly read "default storage class" as the good old "Kubernetes default storage class".

Reply:
That makes sense, but IMO, the rest of the context sufficiently shows that both default storage classes are valid options. :)

Reply:
I buy it.